### PR TITLE
.travis.yml: Build on osx and linux

### PR DESCRIPTION
**Pull request by nkprince007**
Adds support for osx builds

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```

**Comment by nkprince007** [comment]
Atleast the `pytest` ran :P

My bad, setting up `virtualenv` on `osx`
But, I think, it's better if we add a little bit more to the `pytest` configuration.

**Comment by nkprince007** [comment]
Wooohooo :heart_eyes: All green :laughing:

**Comment by abhsag24** [comment]
@nkprince007 add fixes tag in your commit message

**Comment by nkprince007** [comment]
Ok, bro @abhsag24

File: .travis.yml [hunk 1]
```diff
@@ -3,12 +3,24 @@ language: python
...
-  - pip install -r test-requirements.txt
+  - pip3 install -r test-requirements.txt
 
 script:
-  - python setup.py install
+  - python3 setup.py install
   - coala-ci
   - pytest
+
+matrix:
```
Comment (abhsag24) [inline .travis.yml]: why a matrix here and not along with os ?

Comment (nkprince007) [inline .travis.yml]: Nope, since macOS Travis doesn't support we can't use it the other way around

Comment (abhsag24) [inline .travis.yml]: hmm ok

**Comment by abhsag24** [comment]
ack 4ecb0bd

**Comment by abhsag24** [comment]
@nkprince007 Refactor Code is not a nice tag to have since it tellse me nothing about the part of code which got edited. I'll merge anyways since its kind of necessary

**Comment by abhsag24** [comment]
ack bfe8008

**Comment by abhsag24** [comment]
ack bfe8008 gitmate let me merge this one please ! _/\_

**Pull request merged**